core/test/vault-test.ts

```diff
@@ -231,19 +230,19 @@ describe('Sound values are extracted', () => {
             nextPriceChange: new Date('2022-01-23T22:00:00.000Z'),
         },
         'ETH-A': {
-            nextUnitPrice: '1712.2106886',
-            currentUnitPrice: '1712.2106886',
-            nextPriceChange: new Date('2022-09-09T10:00:00.000Z'),
+            nextUnitPrice: '1602.1803800999999',
+            currentUnitPrice: '1602.1803800999999',
+            nextPriceChange: new Date('2022-09-14T13:00:00.000Z'),
         },
         'ETH-B': {
             nextUnitPrice: '1602.1803800999999',
             currentUnitPrice: '1602.1803800999999',
             nextPriceChange: new Date('2022-09-14T13:00:00.000Z'),
         },
         'ETH-C': {
-            nextUnitPrice: '1208.0159951',
-            currentUnitPrice: '1227.067888375',
-            nextPriceChange: new Date('2022-06-13T11:00:00.000Z'),
+            nextUnitPrice: '1602.1803800999999',
+            currentUnitPrice: '1602.1803800999999',
+            nextPriceChange: new Date('2022-09-14T13:00:00.000Z'),
```

Comment:
within several minutes: i did not find the source of side effect:

- checked if we `poke` the oracle - does not seem so
- checked if we overwrite the price - does not seem so
- checked if there's additional modifier for determination of unitPrice - does not seem source

I would have to look longer/deeper to determine the reason

Reply:
Thanks, I think it might be because something else is executed at the time of this test, block number changed and it somehow affects the numbers (but I still can't make sense of it). Will also look into it